go.mod

```diff
@@ -29,38 +29,38 @@ require (
 	google.golang.org/genproto v0.0.0-20200413115906-b5235f65be36 // indirect
 	google.golang.org/grpc v1.28.0
 	gopkg.in/square/go-jose.v2 v2.5.0 // indirect
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/cloud-provider v0.18.0
-	k8s.io/cri-api v0.18.0 // indirect
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
+	k8s.io/cloud-provider v0.18.6
+	k8s.io/cri-api v0.18.6 // indirect
 	k8s.io/klog/v2 v2.3.0
-	k8s.io/kubectl v0.18.0 // indirect
-	k8s.io/kubernetes v1.18.0
+	k8s.io/kubectl v0.18.6 // indirect
+	k8s.io/kubernetes v1.18.5
```

Comment:
why this is 18.5? it should be 18.6 as its the latest release.

Reply:
yep